content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -251,6 +257,10 @@ sudo systemctl restart containerd
 When using kubeadm, manually configure the
 [cgroup driver for kubelet](/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver/#configuring-the-kubelet-cgroup-driver).
 
+Starting with v1.28 and later, you can enable automatic detection of the
```

Comment:
```suggestion
Starting with Kubernetes v1.28 and later, you can enable automatic detection of the
```

Reply:
Aside, but also a bit relevant:
I'm slightly wary of saying “and later” about alpha features that we can't prove will graduate to stable.

Reply:
Fair point, have to agree on that. I now changed all of these to `In Kubernetes {{< skew currentVersion >}}...`

ok @sftim @mikebrow ?

Reply:
given the concern over skew "and later" comments perhaps.. for Kubernetes v1.2.8, you can..